test/test_Time.cpp

```diff
@@ -96,18 +97,54 @@ BOOST_AUTO_TEST_CASE(fromString)
         " vs. converted: " << tzConverted.toString());
     // End time zone check
 
+    base::Time formatNowUTC0 = base::Time::fromString(
+        "2012-06-14--12.05.06Z:001001Z+0 GMT",
```

Comment:
The first 'Z' is already the time zone info. Not sure what format you mean to implement.